includes/reader-revenue/woocommerce/class-woocommerce-connection.php

```diff
@@ -220,9 +220,6 @@ public static function get_contact_from_order( $order, $payment_page_url = false
 		}
 
 		$customer = new \WC_Customer( $user_id );
-		$metadata = [
```

Comment:
Removing this because we only want the registration method to be set if the order results in a new account registration. And it turns out we're already handling that [here](https://github.com/Automattic/newspack-plugin/blob/master/includes/data-events/class-woo-user-registration.php#L50).